_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -98,52 +113,56 @@ On the VM/external host:
 dig istio-pilot.istio-system
 ```
 ```
-    # This should be the same address shown as "EXTERNAL-IP" in 'kubectl get svc -n istio-system istio-pilot-ilb'
-    ...
-    istio-pilot.istio-system. 0	IN	A	10.128.0.5
-    ...
+# This should be the same address shown as "EXTERNAL-IP" in 'kubectl get svc -n istio-system istio-pilot-ilb'
```

Comment:
Use the same address shown...

Reply:
changing to
# Verify you get the same address as shown as "EXTERNAL-IP" in 'kubectl get svc -n istio-system istio-pilot-ilb'

as it's not about using but about checking the addr matches